KM review (part 2) -- Editorial pass on Section 12 (#122)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

# Security Considerations

> Any solution that conveys information used for security purposes, whether such
> information is in the form of Evidence, Attestation Results, Endorsements, or
> Appraisal Policy must support the security properties of confidentiality,
> integrity, and availability.  A conveyance protocol includes the typical
> transport security considerations:
> 
> * end-to-end encryption,
> * end-to-end integrity protection,
> * replay attack prevention, 
> * denial of service protection,
> * authentication, 
> * authorization,
> * fine grained access controls, and
> * logging in line with current threat models and zero trust architectures.
> 

Reviewer: ```suggestion
Any solution that conveys information used for security purposes, whether
such information is in the form of Evidence, Attestation Results,
Endorsements, or Appraisal Policy must support
the security properties of confidentiality, integrity, and availability.  
A conveyance protocol includes the typical transport security considerations:

* end-to-end encryption,
* end-to-end integrity protection,
* replay attack prevention, 
* denial of service protection,
* authentication, 
* authorization,
* fine grained access controls, and
* logging in line with current threat models and zero trust architectures.

```

Any solution that conveys information used for security purposes, whether
such information is in the form of Evidence, Attestation Results,
Endorsements, or Appraisal Policy must support end-to-end integrity protection
and replay attack prevention, and often also needs to support additional
security properties, including:

* end-to-end encryption,
* denial of service protection,

> * authentication,

Reviewer: ```suggestion
* authentication,
* auditing,
```

* fine grained access controls, and

> * logging in line with current threat models and zero trust architectures.

Reviewer: ```suggestion
* logging.
```

Any solution that conveys information used for security purposes, whether
such information is in the form of Evidence, Attestation Results,
Endorsements, or Appraisal Policy must support

> the security properties of confidentiality, integrity, and availability.  

Reviewer: ```suggestion
confidentiality, integrity, and availability.  
```

> Endorsements, or Appraisal Policy must support
> confidentiality, integrity, and availability.  
> A conveyance protocol includes the typical transport security considerations:
> 
> * end-to-end encryption,
> * end-to-end integrity protection,
> * replay attack prevention, 
> * denial of service protection,
> * authentication, 
> * authorization,
> * fine grained access controls, and
> * logging in line with current threat models and zero trust architectures.

Reviewer 2: ```suggestion
Endorsements, or Appraisal Policy must support end-to-end integrity protection
and replay attack prevention, and often also needs to support additional
security properties, including:

* end-to-end encryption,
* denial of service protection,
* authentication,
* fine grained access controls, and
* logging in line with current threat models and zero trust architectures.
```

Any solution that conveys information used for security purposes, whether
such information is in the form of Evidence, Attestation Results,
Endorsements, or Appraisal Policy, needs to support end-to-end integrity protection
Endorsements, or Appraisal Policy must support end-to-end integrity protection
and replay attack prevention, and often also needs to support additional
security protections.  For example, additional means of authentication,
confidentiality, integrity, replay, denial of service and privacy

> protection are needed in many use cases.

Reviewer 3: The "example" restates the list of 'must' be supported security properties. This seems redundant and/or isn't a real example. Might be better to say "must support end-to-end integrity and replay attack prevention, and often needs to support authentication, confidentiality, denial-of-service and privacy protection."
